ledger-core/v2/network/gateway/base.go

```diff
@@ -200,6 +198,8 @@ func (g *Base) createOriginCandidate() error {
 
 func (g *Base) StartConsensus(ctx context.Context) error {
 
+	// one of the nodes has to be in consensus.ReadyNetwork state,
+	// all other nodes has to be in consensus.Joiner
```

Comment:
this comment should be inside `if`, I've spend 5 minutes re-reading the comment to understand meaning until I realized that it describes code inside `if`